.forge/deploy.sh

```
@@ -206,7 +206,9 @@ for INSTANCE_TYPE in "${INSTANCE_TYPES[@]}"; do
         break
     else
         echo "❌  Failed with instance type: $INSTANCE_TYPE"
-        if [ "$INSTANCE_TYPE" != "${INSTANCE_TYPES[-1]}" ]; then
+        # Get the last item in a POSIX-compliant way
+        local last_index=$(( ${#INSTANCE_TYPES[@]} - 1 ))
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)